### ISSUE TITLE
Explore parallel tile generation options

### ISSUE THREAD

**Issue by zstadler** [IsraelHikingMap/Map#42]
Currently, the tile generation process takes more than 24 hours.
Internally, it is separated into 4 phases:
1. Israel Hiking tiles for zoom levels 7-15
2. Trails overlay tiles
3. Israel Hiking tiles for zoom level 16
4. MTB Israel tiles

According to [This Maperitive discussion](https://groups.google.com/forum/#!topic/maperitive/YTvtwWYcRt8), multiple Maperitive installations can be used to run multiple instances in parallel.

**Comment by zstadler** [comment]
Superseded by Incremental Tiles update #108
